README.md

```diff
@@ -668,6 +668,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Krishna Agarwal](https://github.com/MrKrishnaAgarwal)
   - [Krishna Gupta](https://github.com/krishnazest)
   - [Krishna Kumar](https://github.com/Krishnapro)
+  - [Krishna Khajuria](https://github.com/krish40786)
```

Comment:
Please add your name in line 670, after Krishna Gupta.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient